src/Symfony/Component/HttpFoundation/UrlHelper.php

```diff
@@ -21,12 +21,12 @@
 final class UrlHelper
 {
     private $requestStack;
-    private $requestContext;
+    private $contextAware;
 
-    public function __construct(RequestStack $requestStack, RequestContext $requestContext = null)
+    public function __construct(RequestStack $requestStack, RequestContextAwareInterface $contextAware = null)
```

Comment:
For BC reasons, it is mandatory to keep support for passing a RequestContext. Otherwise, this cannot be accepted